From 7f919486919fb083670f66b5d9054a1c5e2179d0 Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Sat, 12 Apr 2014 12:21:47 +0200
Subject: [PATCH] ruby: do not reject ruby versions greater than 1.9

krossruby seems to build fine with them, so do not require just ruby
1.9.x.
---
 ruby/CMakeLists.txt | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/ruby/CMakeLists.txt b/ruby/CMakeLists.txt
index 7826a2f..07ac65d 100644
--- a/ruby/CMakeLists.txt
+++ b/ruby/CMakeLists.txt
@@ -1,9 +1,6 @@
 find_package(Ruby 1.9 QUIET)
 
-if( RUBY_VERSION VERSION_GREATER "1.10" )
-    message(STATUS "Found Ruby: ${RUBY_EXECUTABLE} (found version \"${RUBY_VERSION}\") but only 1.9.x supported")
-    set(RUBY_FOUND False)
-else()
+if(RUBY_FOUND)
     message(STATUS "Found Ruby: ${RUBY_EXECUTABLE} (found version \"${RUBY_VERSION}\")")
 endif()
 
-- 
1.9.1

